lib/jsdom/browser/Window.js

```diff
@@ -642,6 +642,20 @@ function Window(options) {
 
   this.getComputedStyle = function (elt) {
     elt = Element.convert(elt);
+    let pseudoElt = arguments[1];
+    if (pseudoElt !== undefined && pseudoElt !== null) {
+      pseudoElt = webIDLConversions.DOMString(pseudoElt);
+    }
+
+    if (pseudoElt !== undefined && pseudoElt !== null && pseudoElt !== "") {
+      // TODO: Parse pseudoElt
+
+      notImplemented("window.computedStyle(elt, pseudoElt)", this);
```

Comment:
Let's move this after the throw since we do in fact implement that tiny part of the algorithm.